index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+
+                <div id="arrow-ctn">
+                    <img src="icons/indicator.svg" />
+                </div>
+            </div>
+            <div id="works-ctnr">
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
+
+                        <ul class="tech">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+                <div class="work-card">
+                    <div class="bottom-area">
+                        <div class="top">
+                            <h1 class="text">
+                                Multi-post stories<br>
+                                Gain+Glory
+                            </h1>
+                        </div>
```

Comment:
- [ ] Individual tags do not need to be encapsulated in `div` tags as the purpose of `div` is to group the elements. I would highly recommend not wrapping single elements like `<h1>` inside a `div` tag.